### PR TITLE
Bean UI - Update Governance page

### PR DESCRIPTION
**Pull request by uncoolzero**
Expanded our governance page to support older proposals, EBIPs and more

Updated the list of votes table with ENS support

### CODE REVIEW

File: projects/ui/src/functions/ebipdata/ebipdata.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(output),
+        };
+      };
+    };
+
+    return {
+      statusCode: 403
```
Comment (0xalecks) [inline projects/ui/src/functions/ebipdata/ebipdata.ts:58]: Should seems like the wrong response code. 403 is for unauthorized. Did you mean 404 not found, or 400 bad request?

File: projects/ui/src/functions/oldbipdata/oldBipList.ts
Comment (0xalecks) [inline projects/ui/src/functions/oldbipdata/oldBipList.ts:1]: do we need to keep these oldXXXX functions?

**Review by 0xalecks — CHANGES_REQUESTED** [review]

File: projects/ui/src/functions/ebipdata/ebipdata.ts (hunk 1 above)
Comment (uncoolzero) [reply projects/ui/src/functions/ebipdata/ebipdata.ts:58]: updated to 404

File: projects/ui/src/functions/oldbipdata/oldBipList.ts
Comment (uncoolzero) [reply projects/ui/src/functions/oldbipdata/oldBipList.ts:1]: good call, removed the votes as we aren't really using them

File: projects/ui/src/functions/oldbipdata/oldbipdata.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(output),
+        };
+      };
+    };
+
+    return {
+      statusCode: 404
```
Comment (0xalecks) [inline projects/ui/src/functions/oldbipdata/oldbipdata.ts]: this should be a 500 if it's an error

**Review by 0xalecks — APPROVED** [review]

**Pull request merged**